torch/nn/modules/batchnorm.py

```diff
@@ -485,7 +495,25 @@ def forward(self, input: Tensor) -> Tensor:
             else:  # use exponential moving average
                 exponential_average_factor = self.momentum
 
-        need_sync = self.training or not self.track_running_stats
+        r""" 
```

Comment:
Any reason for using rstring instead of comments for the following two doc blocks?

Reply:
hmm not really. They were strings from `_BatchNorm` so I kept them this way when copying.